### PR TITLE
Do not propagate blockchain broadcast errors

### PR DESCRIPTION
**Pull request by fishseabowl**
## Description

Fixes pyrsia/pyrsia#1213

This PR don't propagate blockchain broadcast errors because broadcasting doesn't guarantee other nodes will receive the blocks

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### Code Contributions

- [x] I've built the code `cargo build --all-targets` successfully.
- [x] I've run the unit tests `cargo test --workspace` and everything passes.
- [x] I've made sure my rust toolchain is current `rustup update`.

### CODE REVIEW

File: src/blockchain_service/service.rs [hunk 1]
```diff
@@ -133,9 +133,10 @@ impl BlockchainService {
         buf.append(&mut serialize(&block).unwrap());
 
         for peer_id in peer_list.iter() {
-            self.p2p_client
+            let _ = self
```
Comment (erwin1) [inline src/blockchain_service/service.rs:136]:
```suggestion
            if let Err(e) = self
```

File: src/blockchain_service/service.rs [hunk 2]
```diff
@@ -133,9 +133,10 @@ impl BlockchainService {
...
+                .p2p_client
                 .request_blockchain(peer_id, buf.clone())
-                .await?;
+                .await;
```
Comment (erwin1) [inline src/blockchain_service/service.rs:139]:
```suggestion
                .await {
                log::info!("Failed to send request_blockchain to peer {:?}. Error = {:?}", peer_id, e);
            }
```

**Review by erwin1 — COMMENTED** [review]
I tested it, and it does indeed work around the issue. I would however like to see a log message printed if this happens, so I tried to suggest the changed needed for that.

**Comment by erwin1** [comment]
I hope you didn't mind me committing the change in your PR already :)

**Review by tiainen — APPROVED** [review]

**Comment by fishseabowl** [comment]
> I hope you didn't mind me committing the change in your PR already :)

Thanks for your comments. :)

**Review by erwin1 — APPROVED** [review]

**Pull request merged**